linky.post_update.php

```diff
@@ -93,6 +93,11 @@ function linky_post_update_revisionable(&$sandbox) {
     ])
     ->setDisplayConfigurable('form', TRUE);
 
+  // Update existing fields.
+  $fieldStorageDefinitions['changed']->setRevisionable(TRUE);
```

Comment:
This field doesn't make sense for revisions.

Reply:
works for nodes.